[Marketplace Contribution] Prisma SASE by Palo Alto Networks - Content Pack Update (#30645)
demisto/content: Packs/PrismaAccess/ReleaseNotes/2_1_3.md

#### Playbooks

##### New: SentinelOne - Get Agent Details given Agent ID List

> Given a list of SentinelOne Agents, this playbook is used to fetch a list of Agent IDs and trigger the SentineOne get agent to fetch the details of the endpoint agent where SentinelOne has been installed. The computername/hostname, threatcount and agentdecommissioned fields gleaned from the API response are used for quarantine action.

Reviewer: ```suggestion
Given a list of SentinelOne Agents, this playbook fetches a list of Agent IDs and triggers the SentineOne get agent to fetch the details of the endpoint agent where SentinelOne has been installed. The computername/hostname, threatcount and agentdecommissioned fields gleaned from the API response are used for quarantine action.
```

##### New: Prisma SASE - Sentinelone Integration

> - This playbook is used to ingest threat signals from SentinelOne endpoint , specifically the threat count and agent decommissioned and is used to quarantine the endpoint in Prisma Access. This playbook triggers the SentinelOne Get Agent details playbook and the Prisma-SASE subplaybook.

Reviewer: ```suggestion
This playbook ingests threat signals from SentinelOne endpoint , specifically the threat count and agent decommissioned and quarantines the endpoint in Prisma Access. This playbook triggers the **SentinelOne Get Agent Details** playbook and the **Prisma-SASE** sub-playbook.
```

##### New: Prisma SASE - Sentinelone Integration Subplaybook

> - This Subplaybook is used to fetch the mapping of the computername/hostname to hostID from the CDL logs. This host id is used to quarantine the device if threatCOunt is not equal to 0 and agent is decommissioned.

Reviewer: ```suggestion
This sub-playbook fetches the mapping of the computername/hostname to hostID from the CDL logs. This host ID is used to quarantine the device if threatCOunt is not equal to 0 and the agent is decommissioned.
```
